sdk/storage/azblob/appendblob/client.go

```diff
@@ -326,6 +327,7 @@ func (ab *Client) GetTags(ctx context.Context, o *blob.GetTagsOptions) (blob.Get
 	return ab.BlobClient().GetTags(ctx, o)
 }
 
+// Deprecated: CopyFromURL works only with block blob
```

Comment:
The `// Deprecated` comment must be the last doc comment line.

I think you can also shorten the entire doc comment so the deprecation is more noticeable.  Maybe something like this.
```go
// CopyFromURL
// Deprecated: CopyFromURL works only with block blob
```

Reply:
fixed!